module-apps/application-phonebook/widgets/PhonebookListView.cpp

```diff
@@ -58,7 +57,7 @@ namespace gui
             body->addWidget(item);
 
             // if added item is not visible -> page is full.
-            if (item->visible != true) {
+            if (!item->visible) {
```

Comment:
Should we use `!` or `not`?

Reply:
Good question ;) When modifying existing code, I try to follow author's convention to keep it consistent. When writing my own, I use `!` almost exclusively.